src/Core/src/ImageSources/FileImageSourceService/FileImageSourceService.Windows.cs

```diff
@@ -40,7 +42,27 @@ public partial class FileImageSourceService
 			}
 		}
 
-		BitmapImage GetAppPackage(string filename)
+		static async Task<BitmapImage?> GetImageAsync(string filename)
+		{
+			string fileExtension = Path.GetExtension(filename);
+
+			if (!string.IsNullOrEmpty(fileExtension))
+				return await GetLocal(filename) ?? GetAppPackage(filename);
+
+			var imageExtensions = new List<string> { ".jpg", ".png", ".bmp", ".gif", ".tif" };
+
+			foreach (var imageExtension in imageExtensions)
+			{
+				var filenameExtension = filename + ".scale-100" + imageExtension;
+
+				if (FileSystemUtils.AppPackageFileExists(filenameExtension))
+					return await GetLocal(filenameExtension) ?? GetAppPackage(filenameExtension);
```

Comment:
If we are checking if an app packaged exists and then it does, we don't need the GetLocal().